addons/l10n_jo_edi/views/report_invoice.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <template id="l10n_jo_edi_report_invoice_document" inherit_id="account.report_invoice_document">
+            <xpath expr="//div[@id='qrcode']" position="after">
+                <div t-if="o.l10n_jo_edi_qr" name="qr_code">
+                    <p>
+                        <strong class="text-center">JoFotara QR Code</strong>
+                        <img style="display:block;"
+                            t-att-src="'/report/barcode/?barcode_type=%s&amp;value=%s&amp;width=%s&amp;height=%s'%('QR', o.l10n_jo_edi_qr, 200, 200)"/>
```

Comment:
What about having a method returning all the extra values for the report and formatting already the barcode? That's what we do in mexico and it seems more elegant to me. That way, you reduce the number of computation in the templates (more flexible with the stable version) and it will be easier to debug/add new features on top later